packages/teams-js/src/internal/communication.ts

```diff
@@ -482,6 +529,82 @@ function processMessage(evt: DOMMessageEvent): void {
   }
 }
 
+const processAuthBridgeMessageLogger = communicationLogger.extend('processAuthBridgeMessage');
+
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+function processAuthBridgeMessage(evt: MessageEvent, onMessageReceived: (response: string) => void): void {
+  const logger = processAuthBridgeMessageLogger;
+
+  // Process only if we received a valid message
+  if (!evt || !evt.data || typeof evt.data !== 'object') {
+    logger('Unrecognized message format received by app, message being ignored. Message: %o', evt);
+    return;
+  }
+
+  const { args } = evt.data as MessageResponse;
+  const [, message] = args ?? [];
+  const parsedData = (() => {
```

Comment:
Is there a type we could add for this?

Reply:
Added a simple type with `messageType`. I don't want to move the entire NAA MSAL types over so best to keep it strict to the attributes we'll be using.